config/database.cr

```diff
@@ -1,5 +1,19 @@
 database_name = "avram_dev"
 
+# Adding this struct makes everything work
+#
+# Removing it causes:
+#
+# BUG: trying to upcast Dexter::Formatters::JsonLogFormatter <- Dexter::Formatters::BaseLogFormatter (Exception)
+#   from ???
+#   from ???
+#   from ???
+struct NotEvenUsedAnywhere < Dexter::Formatters::BaseLogFormatter
```

Comment:
This fixes the issue...when it is commented out the compiler bug pops up

Reply:
@jwoertink Did you comment this out when testing?

Reply:
nope. Didn't comment out anything, but I did have to update the repo for crystal 0.29 support. (i.e. fix requires)

Reply:
I believe you need to comment this struct out for the bug to pop up.

Reply:
oh, I read it wrong lol. Ok

Reply:
Alright, commented that struct out, and `./scripts/test` still runs fine. Also:

```
[08:27AM] avram ((619d94f...))$ docker-compose run --rm app crystal spec/
Starting avram_db_1 ... done
.........................................................................*............*.....................................................................................................................................................................................................

Pending:
  Avram::Form .create on failure logs the failure if a logger is set
  Avram::Form .update on failure logs the failure

Finished in 17.41 seconds
284 examples, 0 failures, 0 errors, 2 pending
```

Reply:
Oh awesome! I'll give. it a shot too and close this out 🤞